### DISCUSSION TITLE
Constructor Documentation

### DISCUSSION THREAD

**Discussion by rdennis** [orchidjs/tom-select#139] [Ideas]
Unless I'm missing it, there is no documentation on the first parameter of the `TomSelect` constructor. Looking at the source I see there are a number of accepted types, but all of the examples just use an id selector. It would be helpful to have all of the supported constructor types documented, possibly with examples where that makes sense.

**Comment by oyejorge** [comment]
Just added https://tom-select.js.org/docs/api/#constructor
